### PR TITLE
Add docfiles

### PR DESCRIPTION
**Pull request by veni-vidi-code**
## Summary

#36 

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: docs/migrating_2.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.. currentmodule:: nextcord
+
+.. _migrating_2_0:
+
+Migrating to v2.0
+======================
+
+v2.0 is a big breaking change to this project. Not only is this only a fork now, but with migrating to the new
+discord API a lot of changes were necessary.
```
Comment (redcreeper14385) [inline docs/migrating_2.rst:9]: Capital D :)

**Review by redcreeper14385 — APPROVED** [review]
Everything is good except one capitalisation, otherwise LGTM

File: docs/migrating_2.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.. currentmodule:: nextcord
+
+.. _migrating_2_0:
+
+Migrating to v2.0
+======================
+
+v2.0 is a big breaking change to this project. Not only is this only a fork now, but with migrating to the new
```
Comment (TAG-Epic) [inline docs/migrating_2.rst:8]: This is horribly worded.

File: docs/migrating_to_nextcord.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.. currentmodule:: nextcord
+
+.. _migrating_nextcord:
+
+Migrating to nextcord
+======================
+
+Due to the `original discord.py repository <https://github.com/Rapptz/discord.py>`_ going to read only we decided
+that it would be necessary to fork and keep on developing further. We also wanted to change the name and voted on
```
Comment (TAG-Epic) [inline docs/migrating_to_nextcord.rst:9]: Possibly change it to fork it.

File: docs/whats_new.rst [hunk 4]
```diff
@@ -11,6 +11,16 @@ Changelog
 This page keeps a detailed human friendly rendering of what's new and changed
 in specific versions.
 
+.. _vp2p0p0:
+
+v2.0.0
+--------
+
+The changeset for this version are too big to be listed here, for more information please
```
Comment (TAG-Epic) [inline docs/whats_new.rst:19]: Are should be is here right?

File: docs/whats_new.rst [hunk 5]
```diff
@@ -1184,4 +1194,4 @@ Bug Fixes
 - Mentions are now triggered normally. This was changed due to the way discord handles it internally.
 - Fix issue when a :class:`Message` would attempt to upgrade a :attr:`Message.server` when the channel is
   a :class:`Object`.
-- Unavailable servers were not being added into cache, this has been corrected.
\ No newline at end of file
+- Unavailable servers were not being added into cache, this has been corrected.
```
Comment (TAG-Epic) [inline docs/whats_new.rst:1197]: Why did this change?

File: docs/whats_new.rst [hunk 6]
```diff
@@ -11,6 +11,16 @@ Changelog
 This page keeps a detailed human friendly rendering of what's new and changed
 in specific versions.
 
+.. _vp2p0p0:
+
+v2.0.0
+--------
+
+The changeset for this version are too big to be listed here, for more information please
+see :ref:`the migrating page <migrating_2_0>`. Scince the original discord.py repro ended
```
Comment (TAG-Epic) [inline docs/whats_new.rst:20]: Typo

File: docs/whats_new.rst [hunk 7]
```diff
@@ -11,6 +11,16 @@ Changelog
...
+its work this is thee point where our fork comes in place. Please see
```
Comment (TAG-Epic) [inline docs/whats_new.rst:21]: Typo

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: docs/whats_new.rst (hunk 4 above)
Comment (veni-vidi-code) [reply docs/whats_new.rst:19]: is the same for v1.0

File: docs/whats_new.rst (hunk 5 above)
Comment (veni-vidi-code) [reply docs/whats_new.rst:1197]: it just reflects the amount of lines that changed by inserting the above part

**Review by TAG-Epic — APPROVED** [review]
LGTM

**Pull request merged**